tests/InputFiles/CurvedScalarWave/WorldtubeKerrSchild.yaml

```diff
@@ -134,6 +134,7 @@ Worldtube:
   ExpansionOrder: 1
   Charge: 0.5
   SelfForceOptions:
+    ApplySelfForce: false
     Mass: 0.5
```

Comment:
Not for this PR, but if you plan on adding more options to `SelfForceOptions` it may be worth considering rearranging options so the input file can look like

```yaml
Worldtube:
  SelfForceOptions: None
```

when you want to turn off self-force instead of having to supply a bunch of options that won't be used.

Reply:
Yes, I was thinking about doing that later. Would I have to make an option creatable-class for this?

Reply:
Yes you'd have to make a class that's option creatable, then use that class wrapped in an `Options::Auto<ClassName, Options::AutoLabel::None>` as the type of option to create.